package.json

```diff
@@ -10,9 +10,7 @@
   "scripts": {
     "prebootstrap": "node prebuild.js",
     "test": "lerna run test",
-    "bootstrap": "lerna bootstrap",
-    "lint": "lerna run lint",
-    "lint:js": "lerna run lint",
+    "bootstrap": "lerna bootstrap --npm-client=yarn --force-local",
```

Comment:
This is where this implementation fails. I was able to get the linting to work with the steps above when doing an `yarn install` in each `package` but lerna has some limitations: 
- the bootstrap command is trying to reconcile cross package dependencies 
- this process isn't a straight `yarn install` in each package. 
  - it doesn't install locally referenced files- it only installs external dependencies 
  - `$INIT_CWD` is not localized to each package - it references the monorepo root (this this expected but it would be nice if it had some sort of replacement)

I tried these flags but I don't understand the difference between `bootstrap` and `install` well enough to know if this approach is feasible. Will spend a little more time on this but if it doesn't work I'll just fallback to either having the linting at the monorepo level or pulling everything into each one of the packages.

Reply:
sidenote: i'm going to leave this `--npm-client` flag so we'll probably get some `yarn.lock` updates in this. i can PR this separately if you want.

Reply:
I see that now- I assumed it wasn't set since I got a few `yarn.lock` updates.